build.gradle

```diff
@@ -22,6 +22,10 @@ group = project.maven_group
 tasks.remapJar.dependsOn(shadowJar)
 (tasks.remapJar.input as FileSystemLocationProperty<? extends FileSystemLocation>).set(shadowJar.archivePath)
 
+repositories {
+	maven { url 'http://server.bbkr.space:8081/artifactory/libs-release' }
+}
```

Comment:
We kinda already have a repositories block at the top of the file

Reply:
👀

Reply:
Kara why did you mark this as resolved